src/Uno.UI/Controls/CommandBarNavigationItemRenderer.iOS.cs

```diff
@@ -185,9 +185,13 @@ protected override void OnBeforeArrange()
 
 		protected override Size MeasureOverride(Size availableSize)
 		{
+			//for the same reason expressed below we need to check if iOS sent a wrong availableSize which can be validated
+			// by checking the availableSize.Height
+			_blockReentrantMeasure = availableSize.Height == 0 && _lastAvailableSize?.Height != 0;
```

Comment:
This might set `_blockReentrantMeasure` to `false` when it was `true` (ie it really is a reentrant measure), is that what you want?

Reply:
Was not intentional. Thanks for the comment